src/web/stash/ItemTooltip.tsx

```diff
@@ -14,18 +16,52 @@ export function ItemTooltip({ item }: { item: Item }) {
     return <span class={className}>{item.name}</span>;
   }
 
+  const rangeMap = getRanges(item).reduce(function (
+    map: { [prop: string]: number[] },
+    range
+  ) {
+    if (
+      range.min &&
+      range.max &&
+      range.min !== range.max &&
+      !range.prop.endsWith("-skill") &&
+      !["skill-rand", "charged"].includes(range.prop)
```

Comment:
I was about to say you should move this part of the check inside of the `stats` loop  on  use `stat.type === "other"` instead. It's more robust, and will also handle cases like elemental damage that do have a min and max but aren't a range in the sense this function needs.

But I just realized you should just reuse `checkRange()` in `perfectionScore.ts`, as it already accepts a flexible callback. Something like:

```
if (range.min !== range.max) {
  // TODO: special case for sockets, see perfection score logic.

  checkRange(range, item.modifiers, (_, min, max) => map[stat] = [min,max]);
}
```